test/Test/Lecture1.hs

```diff
@@ -44,6 +44,7 @@ lecture1Spec = describe "Lecture 1" $ do
         it "Negative to zero"  $ subString (-5) 0    "Hello!" `shouldBe` "H"
         it "0 to -1"           $ subString 0 (-1)    "Hello!" `shouldBe` ""
         it "1 to -1"           $ subString 1 (-1)    "Hello!" `shouldBe` ""
+        it "Start > 0"         $ subString 2 4       "Hello!" `shouldBe` "llo"
```

Comment:
Heh, that's awkward that there's no such test in the unit test suite 😅 

Feel free to contribute this test case in a separate PR to the original repository and I'll gladly merge it 🔀